ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -27,7 +27,9 @@ SSHOPTS="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/
 
 function getlogs() {
   echo "### Downloading logs..."
-  scp $SSHOPTS root@$IP:/tmp/artifacts/\*.tar\* ${ARTIFACT_DIR}
+  if ! scp $SSHOPTS root@$IP:/tmp/artifacts/\*.tar\* ${ARTIFACT_DIR} >&/dev/null; then
```

Comment:
There are multiple reasons why this could fail - maybe remove the redirect to /dev/null so it's visible why the SCP failed?

Reply:
That seems reasonable to me - we shouldn't assume a failed SCP means no logs.

Reply:
In general it's true, but there are previous commands in the step that could fail if `SSHOPTS` or `IP` has issues...

Reply:
scp failing does not mean there were no logs, the server may have kernel panicked for all we know. I'd prefer to remove eating stdout, and change the error to say what we know, scp failed.

Reply:
Anyhow, preparing alternative version

Reply:
stderr tells you the problem anyway, it'll say the actual problem (in the linked job, `scp: /tmp/artifacts/*.tar*: No such file or directory`).